src/utils/templateUtils.ts

```diff
@@ -0,0 +1,53 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { Octokit } from '@octokit/rest';
+import { IActionContext, IAzureQuickPickItem } from 'vscode-azureextensionui';
+import { RepoNameStep } from '../commands/createRepo/RepoNameStep';
+import { GitHubOrgListStep } from '../commands/createStaticWebApp/GitHubOrgListStep';
+import { IStaticWebAppWizardContext } from '../commands/createStaticWebApp/IStaticWebAppWizardContext';
+import { createOctokitClient } from '../commands/github/createOctokitClient';
+import { templateReposUsername } from '../constants';
+import { RepoData, RepoResponse } from '../gitHubTypings';
+import { localize } from './localize';
+
+export async function getTemplateReposFromGitHub(context: IActionContext): Promise<RepoData[]> {
+    const client: Octokit = await createOctokitClient(context);
+
+    const allRepositories: RepoResponse = await client.repos.listForUser({
```

Comment:
Would a user be able to see the Microsoft templates from this list?  Or would we have to pass in the url of the Microsoft templates specifically?

Reply:
I'm not sure what you mean by

> Would a user be able to see the Microsoft templates from this list?

Line 19 gets all of the repositories on the https://github.com/staticwebdev account. And then I return only the template repositories on line 23. The only place where the templates are displayed for users are in the quick pick where they select a template.